### PR TITLE
ref(ui): Convert sdkUpdates to Typescript

### PR DESCRIPTION
**Pull request by priscilawebdev**
Convert `sdkUpdates` to Typescript. The component looks like:

image

### CODE REVIEW

**Review by untitaker — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const SDKUpdates = ({event}: Props) => {
+  const {sdkUpdates} = event;
+
+  return (
+    <EventDataSection title={null} type="sdk-updates">
+      {sdkUpdates.map((sdkUpdate, index) => {
+        const suggestion = getSuggestion({suggestion: sdkUpdate, event});
+
+        if (!suggestion) {
+          return null;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx:26]: Does this case happen in practice? An empty DataSection will still consume some vertical space. Should the return be above the EventDataSection to emit no DOM?

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx:26]: I believe this doesn't happen, but I can not guarantee it....better do as you say...thanks 😊

**Review by matejminar — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  return (
+    <EventDataSection title={null} type="sdk-updates">
+      {eventDataSectinContent}
+    </EventDataSection>
+  );
+};
+
+export default SDKUpdates;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx:47]: @priscilawebdev Can we treat acronyms/abbreviations as words to be consistent? So this should be `SdkUpdates`

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx:47]: sure I will update it tomorrow 😉

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx:47]: Done through this PR: https://github.com/getsentry/sentry/pull/20772

**Pull request merged**